addon/mixins/routes/pagination.js

```diff
@@ -36,7 +38,9 @@ export default Ember.Mixin.create({
     controller.setProperties({
       pageNumber: parseInt(this.get('pageNumber')),
       pageSize: parseInt(this.get('pageSize')),
-      totalPages: parseInt(this.get('totalPages'))
+      totalPages: parseInt(this.get('totalPages')),
+      totalRecords: parseInt(this.get('totalRecords')),
+      currentIterationCount: parseInt(this.get('currentIterationCount')),
```

Comment:
I'd prefer that this gets pulled out of a mixin and into a higher order function that returns an extended route at some point, the query params could just get overridden if the route needs additional query params, requiring them to be defined again.

Reply:
Lets add an issue for pulling this out.